docs/metrics.rst

```diff
@@ -41,6 +41,13 @@ Add the following lines to your configuration file e.g. ``airflow.cfg``
     statsd_port = 8125
     statsd_prefix = airflow
 
+If you want to avoid send all the available metrics to StatsD, you can configure a whitelist of prefixes to send only
+the metrics that start with the elements of the list:
+
+.. code-block:: bash
+
```

Comment:
```suggestion
           [core]
```

Reply:
This config is under `[scheduler]`. I'll add that.